docs/tutorials/writing-custom-configuration-audit-policies.md

```diff
@@ -30,7 +30,7 @@ __rego_metadata__ := {
 }
 ```
 
-Note that the `recommended_labels` policy is scoped to the `trivyoperator.policy.k8s.custom` package to avoid naming
+Note that the `recommended_labels` policy must be scoped to the `trivyoperator` package to avoid naming
```

Comment:
maybe also change the package of the example above? I know that it's still valid, but it could be minimized to better  reflect the requirements: `trivyoperator.recommended_labels`

Reply:
done https://github.com/aquasecurity/trivy-operator/pull/2440/commits/b9c59969c8d9106a34989ed1f2aa93168172c876